libs/governance/src/lib/proposals-data-provider/proposals.mock.ts

```diff
@@ -50,7 +65,33 @@ export const marketUpdateProposal: ProposalListFieldsFragment = {
         instrument: {
           code: '',
           product: {
+            __typename: 'UpdateFutureProduct',
             quoteName: '',
+            dataSourceSpecBinding: {
+              __typename: 'DataSourceSpecToFutureBinding',
+              settlementDataProperty: '',
+              tradingTerminationProperty: '',
+            },
+            dataSourceSpecForSettlementData: {
+              __typename: 'DataSourceDefinition',
+              sourceType: {
+                __typename: 'DataSourceDefinitionInternal',
+                sourceType: {
+                  __typename: 'DataSourceSpecConfigurationTime',
+                  conditions: [],
+                },
+              },
+            },
+            dataSourceSpecForTradingTermination: {
+              __typename: 'DataSourceDefinition',
+              sourceType: {
+                __typename: 'DataSourceDefinitionInternal',
+                sourceType: {
+                  __typename: 'DataSourceSpecConfigurationTime',
+                  conditions: [],
+                },
+              },
+            },
```

Comment:
**thought (non-blocking)**: I feel like we're missing a trick with mock generation from the schema. I haven't looked in to it because it works this way, and I suspect part of the problem is everything is a string - but there might be a way of adding in `example` annotations in the schema.

Definitely a thought. Absolutely definitely not a blocking thought.